providers/linux/process_linux.go

```diff
@@ -204,6 +205,33 @@ func (p *process) Capabilities() (*types.CapabilityInfo, error) {
 	return readCapabilities(content)
 }
 
+func (p *process) User() (types.UserInfo, error) {
+	content, err := ioutil.ReadFile(p.path("status"))
+	if err != nil {
+		return types.UserInfo{}, err
+	}
+
+	var user types.UserInfo
+	err = parseKeyValue(content, ":", func(key, value []byte) error {
+		// See proc(5) for the format of /proc/[pid]/status
+		switch string(key) {
+		case "Uid":
+			ids := strings.Split(string(value), "\t")
+			user.Uid = ids[0]
+			user.Euid = ids[1]
+			user.Suid = ids[2]
```

Comment:
This should check the len(ids) >= 3 just to be safe.

Reply:
👍